SROS2_Linux.md

```diff
@@ -116,6 +115,38 @@ Note: You can switch between the C++ (demo_nodes_cpp) and Python (demo_nodes_py)
 
 These nodes are able to communicate because we have created the appropriate keys and certificates for them.
 
+To be able to use the ros2 CLI tools to interact with your secured system, you need to provide it with an override enclave:
+```bash
+export ROS_SECURITY_ENCLAVE_OVERRIDE=/talker_listener/listener
+```
+
+Then use the CLI as usual:
+
+```bash
+ros2 node list
```

Comment:
It would be accessing the ros2 daemon, and return empty list? i think we should recommend here to use `--no-daemon` option to get the node graph via underlying rmw implementation directly. (and else where.)

Reply:
:+1: